### PR TITLE
Alpha Release v0.17.0-alpha

### PR DESCRIPTION
**Pull request by nvukobratTT**

### CODE REVIEW

File: model_demos/README.md [hunk 1]
```diff
@@ -39,11 +39,13 @@ python cv_demos/resnet/pytorch_resnet.py
...
 | [DenseNet](cv_demos/densenet/)                            |   ✔️    |    ✔️    |    ✔️    |        v0.12.3        |
 | [DistilBERT](nlp_demos/distilbert/)                       |   ✔️    |    ✔️    |    ✔️    |        v0.12.3        |
+| [DLA](cv_demos/dla/)                                      |   ✔️    |    ✔️    |    ✔️    |          TBD          |
 | [DPR](nlp_demos/dpr/)                                     |   ✔️    |    ✔️    |    ✔️    |        v0.12.3        |
 | [DLA](cv_demos/dla/)                                      |   ✔️    |    ✔️    |    ✔️    |        v0.15.0-alpha  |
 | [EfficientNet-Lite](cv_demos/efficientnet_lite/)          |   ✘    |    ✘    |    ✔️    |        v0.12.3        |
 | [Falcon-7B](nlp_demos/falcon/)                            |   ✘    |    ✘    |    ✔️    |        v0.12.3        |
 | [FLAN-T5](nlp_demos/flant5/)                              |   ✔️    |    ✔️    |    ✔️    |        v0.12.3        |
+| [FPN](cv_demos/fpn/)                                      |   ✔️    |    ✔️    |    ✔️    |          TBD          |
 | [Fuyu-8B](nlp_demos/fuyu8b/)                              |   ✘    |    ✘    |    ✘    |        v0.12.3        |
```
Comment (milank94) [inline model_demos/README.md:49]:
```suggestion
| [Fuyu-8B](nlp_demos/fuyu8b/)                              |   ✘    |    ✘    |    ✘    |        TBD        |
```

File: model_demos/README.md [hunk 2]
```diff
@@ -66,6 +68,8 @@ python cv_demos/resnet/pytorch_resnet.py
 | [ResNeXt](cv_demos/resnext/)                              |   ✔️    |    ✔️    |    ✔️    |        v0.12.3        |
 | [RetinaNet](cv_demos/retinanet/)                          |   ✔️    |    ✔️    |    ✔️    |        v0.12.3        |
 | [RoBERTa](nlp_demos/roberta/)                             |   ✔️    |    ✔️    |    ✔️    |        v0.12.3        |
+| [SSD300RESNET50](cv_demos/ssd300_resnet50/)               |   ✔️    |    ✔️    |    ✔️    |          TBD          |
```
Comment (milank94) [inline model_demos/README.md:71]:
```suggestion
| [SSD300 ResNet50](cv_demos/ssd300_resnet50/)               |   ✔️    |    ✔️    |    ✔️    |          TBD          |
```

**Comment by anirudTT** [comment]
| Device | u20 | u22 |
|--------|-----|-----|
| E150   | passed |    passed |
| E75    |   passed    | X- Ongoing |
| N150   | passed |  passed   |

**Comment by anirudTT** [comment]
> Device	u20	u22
> E150	passed	passed
> E75	passed	passed
> N150	passed	passed

--- Updated table to show all tests + combinations passing

**Review by anirudTT — APPROVED** [review]
LGTM :) + all tests passed 💯

**Review by milank94 — APPROVED** [review]
LGTM

**Pull request merged**